src/OracleService/OracleService.cs

```diff
@@ -346,7 +346,7 @@ private async Task<(OracleResponseCode, string)> ProcessUrlAsync(string url)
             if (!protocols.TryGetValue(uri.Scheme, out IOracleProtocol protocol))
                 return (OracleResponseCode.ProtocolNotSupported, null);
 
-            using CancellationTokenSource ctsTimeout = new(Settings.Default.Https.Timeout);
+            using CancellationTokenSource ctsTimeout = new(Settings.Default.MaxTaskTimeout);
```

Comment:
`MaxTaskTimeout` is used to remove oracle requests from blockchain. We need to create a new field.